lib/core/src/model.rs

```diff
@@ -767,6 +708,38 @@ impl From<SwapTree> for InternalSwapTree {
     }
 }
 
+/// LNURL-related wrappers
+pub mod lnurl {
```

Comment:
Why are these structures in a different module (lnurl)? Why not just inside the models?

Reply:
Done, moved them to `models.rs`: cc7a206dabf548af5d2c516d76abcb42487dae0d